website/docs/r/ses_identity_notification.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: "aws"
+page_title: "AWS: ses_identity_notification"
+sidebar_current: "docs-aws-resource-ses-identity-notification"
+description: |-
+  Setting SES Identity Notification
+---
+
+# ses_identity_notification
+
+Setting SES Identity Notification for a domain
+
+## Example Usage
+
+```hcl
+resource "ses_identity_notification" "test" {
+  topic_arn = "${aws_sns_topic.example.arn}"
+  notification_type = "Bounce"
+  identity = "${aws_ses_domain_identity.example.domain}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `topic_arn` - (Optional) The Amazon Resource Name (ARN) of the Amazon SNS topic. If the parameter is omitted from the request or a null value is passed, SnsTopic is cleared and publishing is disabled.
```

Comment:
This looks like it was directly copied from the SDK documentation and reads incorrectly in the Terraform documentation. Instead, we should probably update the second sentence to `Can be set to "" (an empty string) to disable publishing.`